### DISCUSSION TITLE
Editor suggestion: Kanban board

### DISCUSSION THREAD

**Discussion by dreamflasher** [standardnotes/forum#2490] [Suggestions]
Would be amazing to have kanban boards in standardnotes!
This seems maintained: https://github.com/riktar/jkanban

**Comment by atmoio** [comment]
Looks cool. But looks like you have to sort of build out the UI for it, like managing tasks and boards. So it wouldn't just be a trivial implementation.

**Comment by dreamflasher** [comment]
Glad you like the idea :) There might be better libraries which support kanban boards, that was just a quick search. But yeah, it's rather rudimentary. 

What are the requirements for libraries that you could use in standardnotes? I understand you want them to be open source and well maintained. Would anything javascript work? Anything electron?

**Comment by dreamflasher** [comment]
For example: https://www.electronjs.org/apps/akiee and https://github.com/thesourorange/electron-kanban (but both seem not well maintained)

https://www.jqwidgets.com/jquery-widgets-documentation/documentation/jqxkanban/jquery-kanban-getting-started.htm

https://github.com/ccllab/just-kanban

https://www.npmjs.com/package/react-advanced-kanban

https://www.npmjs.com/package/vue-kanban

https://www.npmjs.com/package/kanban-board-app

**Comment by atmoio** [comment]
Well, it would need to be web-based, so no Node.js/Electron. And it has to have a clean, mostly non-opinionated UI. But this is extremely hard to find. I'd already looked into kanban before and hadn't found anything ;) This is probably something we'd want to build ourselves at some point, once we have a luxurious amount of resources.

**Comment by dreamflasher** [comment]
Thank you for your detailed feedback, makes sense! And happy to hear that you had looked into it before.

**Comment by kashura** [comment]
Just wanted to add a +1 for this feature. I just sat down to move some stuff over from Notion and ran into this as a big hurdle.

**Comment by JaspalSuri** [comment]
I'm sorry that you had to run into that, as it's pretty frustrating not being able to put everything in one place and keep it truly private. 

Unfortunately, as Mo said, we'll have to find something that strikes a good balance or build some time down the road.

**Comment by tryonlinux** [comment]
@dreamflasher / @kashura ,

I built a simple Kanban Board Editor for Standard Notes. It can be installed via the link below or you can view the source. There are a couple outstanding issues such as mobile formatting, but works pretty well for simple stuff that I use it for.  (see issues tab/project plan on GitHub for more info)

Install Link: https://listed.to/p/BkuMmDC0nU

Github: https://github.com/tryonlinux/kanban-board-sn
